### PR TITLE
FrontEnd : CSS : Transition and transformation (D) - Part 2 (only contains Transitions)

### PR DESCRIPTION
**Pull request by RC2208**
<hr>

## Description 📜

CSS transitions documentation and code. This my 2nd PR on the issue. 
Fixes # #1061
<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev HTML/CSS/JS

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by RC2208** [comment]
https://github.com/girlscript/winter-of-contributing/pull/2295

Here's the 2nd PR please merge both

File: Web_Development/FrontEnd/CSS_Transition_And_Transformation/Transitions.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+css
+div {
+  transition: width 2s, height 4s;
+}
+
+## Selecting all properties <br>
+transition-property takes a special value: `all`. When all is specified, all CSS transition properties will start move.
+
+css
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS_Transition_And_Transformation/Transitions.md:45]: You need to have the codes in code blocks like 

```css
div {
  transition: width 2s, height 4s;
}
```

File: Web_Development/FrontEnd/CSS_Transition_And_Transformation/Transitions.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+* ease-out - Transition effect with slow end
+* ease-in-out - Transition effect with slow start and end
+* cubic-bezier(n,n,n,n) - lets you define your own values in a cubic-bezier function.
+
+
+![transition1](https://user-images.githubusercontent.com/70155541/134521015-728ab242-f1d0-45e8-b47e-b0587d7cd5fd.gif)
+
+Code For different transition prpoerties: <br>
+css
+div {
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS_Transition_And_Transformation/Transitions.md:67]: Codeblocks here too

File: Web_Development/FrontEnd/CSS_Transition_And_Transformation/Transitions.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  transition: width 2s;
+}
+
+#div1 {transition-timing-function: linear;}
+#div2 {transition-timing-function: ease;}
+#div3 {transition-timing-function: ease-in;}
+#div4 {transition-timing-function: ease-out;}
+#div5 {transition-timing-function: ease-in-out;}
+
+div:hover {
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS_Transition_And_Transformation/Transitions.md:80]: Codeblocks here

**Review by code-reaper08 — CHANGES_REQUESTED** [review]
I've left a review, please look into it, add syntax specific codeblocks to all code examples.

**Comment by RC2208** [comment]
> I've left a review, please look into it, add syntax specific codeblocks to all code examples.

Thank you , I forgot to add them.  Added them now!

**Review by code-reaper08 — COMMENTED** [review]
@2024-SANDHYA have a look here, this is the second part of https://github.com/girlscript/winter-of-contributing/pull/2295/

**Comment by 2024-SANDHYA** [comment]
> 
> 
> @2024-SANDHYA have a look here, this is the second part of #2295

Why two parts @RC2208  ? You can include it in one file itself, just divide it into two parts that's it. What do you say @code-reaper08  ?

**Comment by RC2208** [comment]
> > @2024-SANDHYA have a look here, this is the second part of #2295
> 
> Why two parts @RC2208 ? You can include it in one file itself, just divide it into two parts that's it. What do you say @code-reaper08 ?

Hey!I asked 3-4 supervisors and the program manager if I could make 2 seperate PR's since the documentation for both were very long. They allowed it so it can count as 2 PR's for contribution @2024-SANDHYA

**Comment by 2024-SANDHYA** [comment]
> 
> 
> > > @2024-SANDHYA have a look here, this is the second part of #2295
> > 
> > 
> > Why two parts @RC2208 ? You can include it in one file itself, just divide it into two parts that's it. What do you say @code-reaper08 ?
> 
> Hey!I asked 3-4 supervisors and the program manager if I could make 2 seperate PR's since the documentation for both were very long. They allowed it so it can count as 2 PR's for contribution @2024-SANDHYA

Ya that too is accepted but ,okay as you feel you can do that after all, it's open-source, can't restrict you & your creativity ☺️

**Comment by RC2208** [comment]
> > > > @2024-SANDHYA have a look here, this is the second part of #2295
> > > 
> > > 
> > > Why two parts @RC2208 ? You can include it in one file itself, just divide it into two parts that's it. What do you say @code-reaper08 ?
> > 
> > 
> > Hey!I asked 3-4 supervisors and the program manager if I could make 2 seperate PR's since the documentation for both were very long. They allowed it so it can count as 2 PR's for contribution @2024-SANDHYA
> 
> Ya that too is accepted but ,okay as you feel you can do that after all, it's open-source, can't restrict you & your creativity ☺️

Okay 😁

**Review by code-reaper08 — APPROVED** [review]

**Pull request merged**